src/usertypes.ts

```diff
@@ -0,0 +1,16 @@
+import { IUserType } from './interfaces'
+
+export const USERTYPES: IUserType[] = [
+  {
+    name: 'Employee',
+    code: 'E',
```

Comment:
Same as for frequency, we are trying to use human readable codes instead of single letter ones.
```suggestion
    code: 'employee',
```